src/DynamoUtilities/ExceptionHelpers.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace DynamoUtilities.ExceptionHelpers
+{
+
+    /// <summary>
+    /// Exception class thats useful for sending aggregated exceptions and stacktraces to analytics.
+    /// </summary>
+    [Obsolete("For internal use only")]
```

Comment:
Why not just make this internal?

Reply:
Because this type needs to be passed to the Analytics library where it will need to call these methods.

Reply:
internalsvisibleto?